src/server/services/MusicresServices.js

```diff
@@ -0,0 +1,10 @@
+const MediaQuery = require('../models/mediaquery');
+
+class MusicresServices {
+  static async getMusicres(label) {
+    const docs = await MediaQuery.getMusicResByCourse(label);
+    return docs.sort((a, b) => a.title - b.title);
```

Comment:
@maxpluez Isn't it more efficient to let the database sort the results? https://stackoverflow.com/questions/1883264/database-sort-vs-programmatic-java-sort